gce-disk-expand/packaging/debian/control

```diff
@@ -0,0 +1,16 @@
+Source: gce-disk-expand
+Section: admin
+Priority: optional
+Maintainer: Google Cloud Team <gc-team@google.com>
+Build-Depends: debhelper (>= 9),
+               dh-systemd (>= 1.5),
+Standards-Version: 3.9.8
+Homepage: https://github.com/GoogleCloudPlatform/compute-image-packages
+
+Package: gce-disk-expand
+Architecture: all
+Depends: parted,
+         ${misc:Depends}
+Conflicts: cloud-guest-utils, cloud-initramfs-growroot
```

Comment:
I'm thinking we can not conflict with cloud-guest-utils since it doesn't directly do anything. It installs growpart but nothing calls it. We conflicted with it before because the version of growpart we were using was newer than the packaged version from distros usually was. We should still conflict with cloud-initramfs-growroot (which should be EOL anyway).